.circleci/config.yml

```diff
@@ -21,9 +21,26 @@ executors:
   deploy-executor:
     docker: # todo: create new image endpoint, add this in admin tab for faster starts
     - image: *ci_support_image
-    working_directory: ~/repo/pepper-apis  
+    working_directory: ~/repo/pepper-apis
 
 commands:
+  install-tcell:
+    steps:
+      - run:
+          name: Download tcell files from bucket
```

Comment:
Is the plan to simply check tcell files into source? So we don't need this CircleCI command anymore?

Reply:
In the plan-of-the-moment, we'd just grab them from a bucket.  In which case we can delete the files here :)